src-exe/Controller/Api.hs

```diff
@@ -116,6 +116,7 @@ type ControllerApi m =
   :<|> "v2" :> "queue" :> Capture "pkgname" PkgN :> Capture "idxstate" PkgIdxTs :> Get '[JSON]  QEntryRow
   :<|> "v2" :> "queue" :> Capture "pkgname" PkgN :> Capture "idxstate" PkgIdxTs :> ReqBody '[JSON] QEntryUpd :> Put '[JSON] QEntryRow
   :<|> "v2" :> "queue" :> Capture "pkgname" PkgN :> Capture "idxstate" PkgIdxTs :> DeleteNoContent '[JSON] NoContent
+  :<|> "v2" :> "users" :> "name"                 :> Capture "username" UserName :> Get '[JSON] UserPkgs
```

Comment:
this results in `GET /api/v2/users/name/{username}`

we want instead:

 `GET /api/v2/users/{username}`

So change this to:

```
:<|> "v2" :> "users" :> Capture "username" UserName :> Get '[JSON] UserPkgs
```